src/main/host/syscall_handler.c

```diff
@@ -165,6 +165,12 @@ static void _syscallhandler_post_syscall(SysCallHandler* sys, long number,
         scr = syscallhandler_##s(sys, args);                                   \
         _syscallhandler_post_syscall(sys, args->number, #s, &scr);             \
         break
+#define HANDLE_NEW(s)                                                              \
```

Comment:
Probably deserves a more descriptive name and/or a comment. `HANDLE_RUST`?

Reply:
Done.